azkaban-common/src/test/java/azkaban/executor/mail/DefaultMailCreatorTest.java

```diff
@@ -95,7 +95,8 @@ public void createErrorEmail() throws Exception {
     executableFlow.setStatus(Status.FAILED);
     assertTrue(mailCreator.createErrorEmail(
         executableFlow, message, azkabanName, scheme, clientHostname, clientPortNumber));
-    assertEquals("Flow 'mail-creator-test' has failed on unit-tests", message.getSubject());
+    assertEquals("Flow 'mail-creator-test' has failed on unit-tests",
```

Comment:
You're right - it shouldn't be two lines. I had added a change here and then removed it, but forgot to remove the newline. Thanks!